pkg/workflows/create_prep.go

```diff
@@ -0,0 +1,41 @@
+package workflows
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+// CreateNamespaceIfNotPresent creates the namespace on the cluster if it does not already exist.
+func CreateNamespaceIfNotPresent(ctx context.Context, namespace, kubeconfig string, clientFactory interfaces.ClientFactory) error {
```

Comment:
Nit: can we find a different place for this?
Im ok if you move it in a separate pr

Reply:
i'll do it in another pr